docs/src/guide/extensions_networking.md

```diff
@@ -23,6 +23,28 @@
 {{#include ../../../examples/extensions/vars-extensions-cilium.auto.tfvars:4:}}
 ```
 
+If you are looking to replace flannel with cilium, you can use the cluster-addons module to remove flannel extension and kube-proxy (Optional).
```

Comment:
Since Cilium helm values have changed recently, add a note to say from which versions of Kubernetes and Cilium this has been tested.